### ISSUE TITLE
Request: Change download speed limit and pause download for x min

### ISSUE THREAD

**Issue by Benni-chan** [hpsu/nzbget-chrome#17]
would be nice, if we could set a speed limit for downloading from the popup (used to this pretty often with sabnzbd)
also, i'm missing the functionality to pause only for x min (e.g. pause downloads for 30 min, because i need the bandwith, but might forget to reactivate the downloads)

**Comment by xelfer** [comment]
Agreed, would like this feature as it was probably the number 1 thing I used in sabconnect++ addon
